Ryujinx.Graphics.Vulkan/IdList.cs

```diff
@@ -80,8 +80,16 @@ public bool TryGetValue(int id, out T value)
 
             try
             {
-                value = _list[id];
-                return value != null;
+                if (id >= 0 && id < _list.Count)
```

Comment:
```suggestion
                if ((uint)id < (uint)_list.Count)
```
Should be a little bit faster since it does one comparison rather than 2.

Reply:
Sorry I didn't know I could just "Commit suggestion".